google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -20,7 +20,7 @@
   <description>
     <![CDATA[<html>
       <p>This plugin integrates Google Cloud Platform into your IDE development workflows.</p>
-      <p>The following features are all available in IntelliJ IDEA. As of version 18.5.1, <a href="https://github.com/GoogleCloudPlatform/google-cloud-intellij/blob/master/README.md#supported-platforms">some of these features</a> are available in other JetBrains IDEs:</p>
+      <p>The following features are all available in IntelliJ IDEA. As of version 18.5.1, Cloud Storage and Cloud Source Repositories integration are available in PyCharm:</p>
```

Comment:
what do you think of? 
.., **but only** Cloud Storage and ..

Reply:
that sgtm

Reply:
got it.